cern_access/indico_cern_access/util.py

```diff
@@ -183,6 +221,21 @@ def add_access_requests(registrations, data, state, nonces):
     for registration in registrations:
         create_access_request(registration, state, data[registration.id]['$rc'],
                               nonces[generate_access_id(registration.id)])
+        # save the accompanying persons' reservation codes and nonces
+        _, accompanying_persons = get_accompanying_persons(registration, get_last_request(registration.event))
```

Comment:
```suggestion
        accompanying_persons = get_accompanying_persons(registration, get_last_request(registration.event))[1]
```

in particular, do not overwrite `_` which is used for i18n (we usually use `__` when we need such a dummy var) - but here just getting the item directly works perfectly fine